.circleci/config.yml

```
@@ -15,111 +15,147 @@ workflows:
             - amd64_build
           filters:
             branches:
-              ignore: "rel/nightly"
+              ignore:
+                - "hotfix/*"
```

```suggestion
                - "/hotfix\/.*/"
```
For regexes we need to add the forward slashes. https://circleci.com/docs/2.0/configuration-reference/